docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,816 @@
+Also, it's refreshing to see that the standard MUI table components are adequate for many of you.
+
+### What are you building?
+
+<img src="/static/blog/2021-survey/18.png" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" alt="Dashboard admin applications: 26.3%; Enterprise applications: 25.9%; A custom design system: 9.7%; Landing page: 8.1%; E-commerce application: 7.8%; Personal website/portfolio: 6.9%; CMS: 6.1%; Just playing with tech: 4.6%; Blog: 2.8%; Other: 1.4%" />
+
+Enterprise, dashboard applications, and design systems continue to be in the top 3, similar to last year.
+What's changed is that developers are now using MUI more to build e-commerce apps and portfolios, which is pretty cool.
+We'd love to see them! Feel free to share in the ["Who's using Material UI?" issue](https://github.com/mui/material-ui/issues/22426).
+
+### What delivery mechanisms are you using?
+
+<img src="/static/blog/2021-survey/19.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Single-page app (Create React App, etc): 74.7%; Server-side rendered website (Next.js, Gastby, etc): 20.8%; Desktop app (Electron, etc): 3.5%; Native mobile app (Cordova, etc): 0.6%; Other: 0.4%" />
+
+<p class="blog-description">1509 out of 1589 answered.</p>
+
+### What type system are you using?
+
+<img src="/static/blog/2021-survey/20.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="TypeScript: 63.8%; None: 18%; prop-types: 16.6%; Flow: 1.4%; Other: 0.2%" />
+
+<p class="blog-description">1501 out of 1589 answered.</p>
+
+These results are very different from last year—Typescript just jumped off and it's now the primary type system used out there.
+It seems reasonable to expect more people to start using type systems.
+
+### Which framework are you using, if any?
+
+<img src="/static/blog/2021-survey/21.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Create React App: 62.4%; Next.js: 21.9%; Custom webpack: 10.7%; None: 2.6%; Gatsby: 0.9%; Other: 1.5%" />
+
+<p class="blog-description">1497 out of 1589 answered.</p>
+
+This year we saw a considerable bump in Next.js usage, which is not surprising due to its increasing popularity and the advancements it gained in 2021.
+
+### What styling solution are you using?
+
+<img src="/static/blog/2021-survey/22.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="MUI Core v4 (JSS): 45%; Styled components: 37.9%; Emotion: 30.2%; SASS: 20.8%; CSS modules: 18.9%; Vanilla CSS: 17.6%; Tailwind CSS: 9.1%; Stitches: 0.4%; Other: 1.2%" />
+
+<p class="blog-description">1492 out of 1589 answered.</p>
+
+As the MUI Core v5 release is relatively recent, we expect to see many developers still using JSS, and there will probably always be some of you who will prefer it.
+
+But the growth of Emotion and styled-components here is noteworthy, because they are what our new styling solution is based on.
```

Comment:
I had asked emotion about the official capitalization of their header, and it seems to be this one https://github.com/emotion-js/emotion

```suggestion
But the growth of emotion and styled-components here is noteworthy because they are what our new styling solution is based on.
```